pkg/scheduler/frameworkext/framework_extender.go

```diff
@@ -82,9 +83,10 @@ type frameworkExtendedHandleImpl struct {
 	koordinatorClientSet             koordinatorclientset.Interface
 	koordinatorSharedInformerFactory koordinatorinformers.SharedInformerFactory
 	sharedListerAdapter              SharedListerAdapter
+	waitingForLeader                 chan struct{}
 }
 
-func NewExtendedHandle(options ...Option) ExtendedHandle {
+func NewExtendedHandle(waitingForLeader chan struct{}, options ...Option) ExtendedHandle {
```

Comment:
DO NOT directly add parameter here. Please custom new `Option`

Reply:
fixed, use another comment's solution